lib/spack/spack/installer.py

```diff
@@ -1730,8 +1737,14 @@ def _push_task(self, task: BuildTask) -> None:
             return
 
         # Remove any associated build task since its sequence will change
+        def ord(num):
+            suffixes = {"1": "st", "2": "nd", "3": "rd"}
+            d = str(num)[-1]
+            suffix = suffixes[d] if d in suffixes and num % 100 not in [11, 12, 13] else "th"
+            return f"{num}{suffix}"
+
         self._remove_task(task.pkg_id)
-        desc = "Queueing" if task.attempts == 0 else "Requeueing"
+        desc = "Queueing" if task.attempts <= 1 else f"Requeueing ({ord(task.attempts)} time)"
```

Comment:
Why would it be `Queueing` for the first two times around and then `Requeueing` after that?

Reply:
It isn't the first two times.  It is the first time (see outputs pasted in the description).  The constructor initializes the value to `0` so that the initial call to `_update()` sets the value of the queued task to `1`.

Reply:
Done and confirmed the result is unchanged.